### DISCUSSION TITLE
RETRO: 2023/12/12

### DISCUSSION THREAD

**Discussion by H1s0kaaa** [Willypai/devopsProject#16] [General]
1. what went well?
2. what could be improved?
3. what didn't work?
4. what shall we focus?

**Comment by H1s0kaaa** [comment]
1.
The website has been set up successfully.
2.
More spike activities could be added.
3.
None I can think of at the moment.
4.
featuring our react local environment into our repo.

**Comment by b11009061** [comment]
1.今天進度神速，我們將我們的專案成品上傳到了github上，雖然過程遇到了點小問題，但還是成功解決了
2.可能我們目前所使用到的一些工具都太過簡單，未來我們若能再增添功能，希望可以學習更多工具的使用
3.減少使用gpt的次數，多自己上網找尋答案
4.我覺得其實也可以再讓背景更漂亮一點，原神，啟動~
